### PR TITLE
V1.3.1

### PR DESCRIPTION
**Pull request by vidhim**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

File: jenkins-build-pack-website/Jenkinsfile [hunk 1]
```diff
@@ -628,10 +628,11 @@ def updateCloudFrontPolicy(String service, String s3Bucket){
 
 	def bucketPolicy = loadBucketPolicy(s3Bucket)
 	def statements = bucketPolicy.Statement
+	def CFID = getCloudFrontDetails(environment, service)
 
-//	def principal = ["AWS": env.CLOUDFRONT_OAI]
+	//	def principal = ["AWS": env.CLOUDFRONT_OAI]
 	//def principal = ["AWS": "*"]
-	def principal = "*"
+	def principal = {"AWS": "arn:aws:iam::cloudfront:user/CloudFront Origin Access Identity ${CFID[0]}"}
```
Comment (deepusundar) [inline jenkins-build-pack-website/Jenkinsfile]: Validate that this is CF origin access identity and not Distribution Id

File: jenkins-build-pack-website/Jenkinsfile [hunk 2]
```diff
@@ -646,7 +647,7 @@ def updateCloudFrontPolicy(String service, String s3Bucket){
 		new_policy.put('Effect',"Allow")
 		new_policy.put('Action',"s3:GetObject")
 		new_policy.put('Principal', principal)
-		new_policy.put('Resource',"arn:aws:s3:::$s3Bucket/$service/*")
+		new_policy.put('Resource',"arn:aws:s3:::$s3Bucket/*")
```
Comment (deepusundar) [inline jenkins-build-pack-website/Jenkinsfile]: I hope s3 bucket per service is not implemented yet in OSS.  Why the $service is removed from resource ?

File: platform_login/index.js [hunk 3]
```diff
@@ -52,7 +52,7 @@ module.exports.handler = (event, context, callback) => {
 			}
 
 			var authenticationData = {
-				Username : event.body.username,
+				Username : event.body.username.toLowerCase(),
```
Comment (deepusundar) [inline platform_login/index.js]: I hope this is not a change related to CAPI-1252

File: platform_login/index.js [hunk 4]
```diff
@@ -64,7 +64,7 @@ module.exports.handler = (event, context, callback) => {
 			var authenticationDetails = new AWSCognito.AuthenticationDetails(authenticationData);
 			var userPool = new AWSCognito.CognitoUserPool(poolData);
 			var userData = {
-				Username : event.body.username,
+				Username : event.body.username.toLowerCase(),
```
Comment (deepusundar) [inline platform_login/index.js]: I hope this is not a change related to CAPI-1252

File: jenkins-build-pack-website/Jenkinsfile [hunk 5]
```diff
@@ -183,7 +183,7 @@ node ()  {
 
 					def url = 'n/a'
 					if (create_cloudfront_url == true) {
-						updateCloudFrontPolicy(service, s3ProdBucket)
+						updateCloudFrontPolicy(service, s3ProdBucket, current_environment)
```
Comment (deepusundar) [inline jenkins-build-pack-website/Jenkinsfile]: Make the naming convention consistent.  Should follow Camel case for var names

Comment (deepusundar) [reply jenkins-build-pack-website/Jenkinsfile]: Also just put 'environment' instead of current_environment

File: platform_login/index.js (hunk 3 above)
Comment (vidhim) [reply platform_login/index.js]: This is not a change related to CAPI-1252

File: platform_login/index.js (hunk 4 above)
Comment (vidhim) [reply platform_login/index.js]: This is not a change related to CAPI-1252

File: jenkins-build-pack-website/Jenkinsfile (hunk 2 above)
Comment (suryajak) [reply jenkins-build-pack-website/Jenkinsfile]: This is correct, we don't need $service in OSS.

File: jenkins-build-pack-website/Jenkinsfile (hunk 1 above)
Comment (suryajak) [reply jenkins-build-pack-website/Jenkinsfile]: @deepusundar  is correct. This needs to be CF origin access identity. you can get it from aws cli:

https://docs.aws.amazon.com/cli/latest/reference/cloudfront/list-cloud-front-origin-access-identities.html

where you will need to look up using comment field. comment field has format {env-prefix}-origin_access_identity

Comment (suryajak) [reply jenkins-build-pack-website/Jenkinsfile]: def outputStr = sh (
                        script: "aws cloudfront list-cloud-front-origin-access-identities --output text \
									--query \"CloudFrontOriginAccessIdentityList.Items[?Comment=='${configLoader.env_name_prefix}-origin_access_identity'].{j: Id}\"",
                        returnStdout: true
                    ).trim()

                echo outputStr

Comment (suryajak) [reply jenkins-build-pack-website/Jenkinsfile]: You can see the values in the console here: https://console.aws.amazon.com/cloudfront/home?region=us-east-1#oai:

File: jenkins-build-pack-website/Jenkinsfile (hunk 5 above)
Comment (supritat) [reply jenkins-build-pack-website/Jenkinsfile]: As mentioned variable is changed from "current_environment" to "environment"

**Pull request merged**